ui.frontend/html/src/components/App.jsx

```diff
@@ -6,12 +6,12 @@ import BasicWidgetTestView from './BasicWidgetTestView'
 import ConfigurationWizardView from './ConfigurationWizardView'
 import MainView from './MainView'
 import React from 'react'
-import StartSessionWizardView from './StartSessionWizardView'
+import ShareProjectView from './ShareProjectView'
 
 const viewComponents = {
   [views.MAIN]: MainView,
-  [views.ADD_CONTACT]: AddContactView,
-  [views.START_SESSION_WIZARD]: StartSessionWizardView,
+  [views.ADD_CONTACT_PAGE]: AddContactView,
+  [views.SHARE_PROJECT_PAGE]: ShareProjectView,
   [views.CONFIGURATION_WIZARD]: ConfigurationWizardView,
```

Comment:
Now it seems a bit weird that this is still named 'Wizard'. How about either also adjusting this as part of this PR or opening an issue to refactor this later on?

Reply:
Thanks, I changed the name of the constant